docs/platform/what-is-tizen/versions/tizen-6-0-m2.md

```diff
@@ -0,0 +1,104 @@
+# Tizen 6.0
+
+The second milestone (M2) release of Tizen 6.0 was announced in October 2020.
+Tizen is an open-source operating system (OS) maintained by Samsung and is developed and optimized for smart devices since 2012.
+
+The key features of Tizen 6.0 are the enhancement of development environment, core and AI component.
+
+## Enhancement of Development Environment and Core Component
+
+### Tizen.NET
+
+Tizen suppports C# development environment based on .NET Core for application development productivity.
+In Tizen 6.0 the latest .NETCore 3.1 and the latest version of cross-platform Xamarin is supported.
+
+There is a advance in the application start-up time and memory usage reduction.
+and "XAML Hot Reload" beta service is supported as well.
+
+- **.NET Core** :
+Latest .NET Core 3.1(LTS)
+Optimized performance
+Diagnostics tools
+
+- **TizenFX** :
+Tizen platform-specific API set
+API level 8 Released
+
+- **Xamarin** :
+Latest Xamarin.Forms (4.8.0) & Xamarin.Essentials (1.5.3) are available.
+TV UI Controls & CircularUI : New UI controls for Smart TVs and Smart Watches.
+XAML Hot Reload : Boosting Developer productivity
+
+### Web framework
+
+Tizen 6.0 provides a new user experience through device-to-device connectivity and interworking.
+
+- Web-based device various offloading services
+
+Tizen 6.0 provides the multi-device offloading service using web technology.
+
+The complexity of applications such as A.I and AR/VR increases in web side, maximization of computational performance within web contents is needed.
```

Comment:
Tizen 6.0 provides a new user experience through device-to-device connectivity and interworking.

- Web-based device various offloading services

**Tizen 6.0 provides a multi-device offloading service using Web technology. The complexity of applications such as A.I and AR/VR increases in web side, maximization of computational performance within web contents is needed.**

this paragraph is not clear.

Reply:
I changed as follows: 
"As the complexity of applications in the web environment increases, a method of maximizing computational power is needed.

Tizen 6.0 provides 'multi-device offloading service' technology using multiple devices."